app/controllers/country_controller.rb

```diff
@@ -105,7 +105,7 @@ def load_essential_vars
   private 
 
   def flag_path country_name
-    country_name = country_name.downcase.gsub(' ', '-')
-    ActionController::Base.helpers.image_url("flags/#{country_name}.svg")
+    file_string = country_name.downcase.gsub(' ', '-').gsub(',', '')
```

Comment:
Instead of chaining `.gsub`, what you could do is have a hash of characters to replace, like: 
```
characters_to_replace = {
 ' ' => '-',
',' => ''
}
```
and then loop through that hash, so something like:
```
name = country_name.downcase
characters_to_replace.each do |char, replacement| 
 name.gsub!(char, replacement)
end
```
so in future if there's a new character we need to replace (that's non-ASCII, you'll need I18n to replace that), we can just add it to the `characters_to_replace` hash

Reply:
Alternatively you can pass `gsub` a block and put conditional logic in there.